sync/sync-api/src/main/java/com/duckduckgo/sync/api/SyncCrypto.kt

```diff
@@ -17,7 +17,7 @@
 package com.duckduckgo.sync.api
 
 interface SyncCrypto {
-    fun encrypt(text: String): Result<String>
+    fun encrypt(text: String): String
```

Comment:
How do you want to model errors here? throwing exception instead?

We should document api methods

Reply:
Discussed offline, we’ll move to `Result` in another PR